pytest.ini

```diff
@@ -1,5 +1,5 @@
 [pytest]
-addopts = --color=yes --durations=5 --showlocals --verbose
+addopts = --cov=./mlflow --cov-report html --color=yes --durations=5 --showlocals --verbose
```

Comment:
Heads up @harupy this line seems to be preventing breakpoints in PyCharm from working when debugging a test:
https://stackoverflow.com/a/40906766/13837474
Not too much of a problem because I can just temporarily remove these arguments when iterating. But do we have bandwidth to fix this?
cc @dbczumar